Doc/library/tk.rst

```diff
@@ -22,12 +22,13 @@ consult the Tk documentation, and occasionally the Tcl documentation.
 classes.
 
 :mod:`tkinter`'s chief virtues are that it is fast, and that it usually comes
-bundled with Python. Although its standard documentation is weak, good
-material is available, which includes: references, tutorials, a book and
-others. :mod:`tkinter` is also famous for having an outdated look and feel,
-which has been vastly improved in Tk 8.5. Nevertheless, there are many other
-GUI libraries that you could be interested in. For more information about
-alternatives, see the :ref:`other-gui-packages` section.
+bundled with Python. Although its standard documentation is weak, good material
+is available, which includes: references, tutorials, a book and others.
+:mod:`tkinter` is also famous for having an outdated look and feel, which has
+been vastly improved in Tk 8.5. Nevertheless, there are many other GUI libraries
+that you could be interested in. The Python wiki lists several of these
+alternative `GUI frameworks and tools
+<https://wiki.python.org/moin/GuiProgramming>`_.
```

Comment:
It looks like most of the change here is just reflowing the text, which makes it harder to review.  Could you please revert the reflow?